File: swift/Sources/FantasticKernelBridge/EgressRules/Silent.swift
// `silent` — the default egress rule: present no credential (today's wire shape).

extension EgressRules {
    /// Attach nothing to outbound calls — the back-compat default and what every
    /// non-credential-bearing policy (allow_all / deny_inbound) presents.
    public struct Silent: EgressRule {
        public init() {}
        public func credential() -> String? { nil }
    }
}

File: rust/crates/bundles/fantastic-kernel-bridge/src/authorizer/egress/silent.rs
//! `silent` — the default egress rule: present no credential (today's wire shape).

use super::super::EgressRule;

/// Attach nothing to outbound calls — the back-compat default and what every
/// non-credential-bearing policy (allow_all / deny_inbound) presents.
pub struct Silent;

impl EgressRule for Silent {
    fn credential(&self) -> Option<String> {
        None
    }
}
